experiments/experiments/experiment_one_of_many_services_degradation.rb

```diff
@@ -11,13 +11,13 @@
   experiment_name: "One of Many Services Degradation Experiment",
   resource_name: "protected_service",
   degradation_phases: [Semian::Experiments::DegradationPhase.new(healthy: true)] * 1 +
-                      [Semian::Experiments::DegradationPhase.new(latency: 4.95)] * 10 + # Most requests to the target service will timeout
+                      [Semian::Experiments::DegradationPhase.new(latency: 9.95)] * 10 + # Most requests to the target service will timeout
```

Comment:
I had increased the timeout to 10 seconds at some point, and forgot to change that